linux_pipeline/Jenkinsfile_rhel_kernel_validation

```diff
@@ -172,3 +172,29 @@ stage ("Send Email Report"){
         }
     }
 }
+
+stage ("Trigger Hotfix"){
+    node ("jenkins-meta-slave") {
+        if (!sendMail) {
+            return
+        }
+        def TestDistros = ""
+        dir ("trigger-hotfix-${env.BUILD_NUMBER}-${env.BRANCH_NAME}") {
+            stashList.each() {
+                unstash it
+                VersionLog = readFile("${it}_lis_check.log")
+                if (VersionLog.contains("built-in driver")) {
+                    def Kernelversion = (VersionLog =~ "Running kernel version: (.*).x86_64")[0][1]
+                    TestDistros+="${it}=https://${STORAGE_ACCOUNT}.blob.core.windows.net/kernel/kernel-${Kernelversion}.x86_64.rpm,https://${STORAGE_ACCOUNT}.blob.core.windows.net/kernel/kernel-devel-${Kernelversion}.x86_64.rpm"+"\n"
```

Comment:
Folder for the storage blob should be used. All the rpm will be under a folder (named 'errata-kernel-version'). Build pipeline will use this folder name to download the rpm from blob.

Reply:
Changed the logic to pass "distro"="kernel-version" , since in storage blob a folder will be created with the detected kernel version